Applications/WebPortal/WebApp/tests/profile_page_tester/source/index.tsx

```diff
@@ -77,7 +77,7 @@ class TestApp extends React.Component<Properties, State> {
   }
 
   public componentDidMount() {
-    const tempIdentity =  new Nexus.AccountIdentity();
+    const tempIdentity = this.state.identity.clone();
```

Comment:
Rename this from ```tempIdentity```. Can rename it to ```identity``` if you'd like or ```testIdentity```. The issue is that temporary implies this will be local or not used for very long, however, you're updating the state of the identity to this variable so it's not actually temporary.